pyflakes/test/test_checker.py

```diff
@@ -152,10 +152,6 @@ def test_type_comment_without_whitespace(self):
         ret = self._collect('x = 1 #type:int')
         self.assertSetEqual(ret, {(ast.Assign, ('#type:int',))})
 
-    def test_type_comment_starts_with_word_ignore(self):
-        ret = self._collect('x = 1 # type: ignore[T]')
-        self.assertSetEqual(ret, {(ast.Assign, ('# type: ignore[T]',))})
```

Comment:
so it's more clear of the change here -- can this be updated with the new behaviour?

Reply:
Done